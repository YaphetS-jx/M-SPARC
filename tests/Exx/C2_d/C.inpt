# Test: C #
CELL: 18 18 18
FD_GRID: 20 20 20 
FD_ORDER: 12
BC: D D D
#EXCHANGE_CORRELATION: LDA_PW
EXCHANGE_CORRELATION: GGA_PBE
ELEC_TEMP_TYPE: fd
SMEARING: 0.003675
#SMEARING: 0.3166810501
TOL_SCF: 1e-4
NSTATES: 6
PRINT_FORCES: 1
PRINT_ATOMS: 1

EXX_FLAG: 1
