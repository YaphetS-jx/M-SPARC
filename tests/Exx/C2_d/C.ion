#=========================
# format of ion file
#=========================
# ATOM_TYPE:   <atom type name> 
# PSEUDO_POT:  <path/to/pseudopotential/>
# N_TYPE_ATOM: <num of atoms of this type>
# ATOMIC_MASS: <mass of atom of this type> #(optional, for MD only)
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX: #(optional)
# <xrelax> <yrelax> <zrelax>
# ...

# this is a comment

ATOM_TYPE:   C                             # atom type 
N_TYPE_ATOM: 2                             # number of atoms of this type
PSEUDO_POT:  C.pot   # pseudopotential
COORD:                                     # coordinates follows
 8 9 9 
 10 9 9 