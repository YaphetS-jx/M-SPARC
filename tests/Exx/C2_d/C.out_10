***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 04-Nov-2020  Start time: 11:01:04                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 20.000000 20.000000 
FD_GRID: 11 11 11
FD_ORDER: 12
BC: D D D
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003675000
CHEB_DEGREE: 14
NSTATES: 6
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-04
TOL_POISSON: 1.00E-06
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/C2_d/C.out_10
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  1.818182 (Bohr)
Output printed to                  :  ../tests/Exx/C2_d/C.out_10
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  10.91 10.91 10.91
Number of atoms of type 1          :  2
Estimated total memory usage       :  11.15 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.4082536019E+00        1.394E-01        0.060
2            -5.4121329998E+00        8.411E-02        0.019
3            -5.4132721786E+00        2.713E-02        0.025
4            -5.4133486488E+00        3.740E-03        0.020
5            -5.4133559333E+00        1.579E-03        0.040
6            -5.4133568508E+00        2.100E-04        0.018
Total number of SCF: 6     
1            -5.4133567743E+00        6.800E-05        13.275
2            -5.4133567573E+00        4.070E-05        3.826
3            -5.4133564184E+00        7.022E-06        3.894
Total number of SCF: 3     
