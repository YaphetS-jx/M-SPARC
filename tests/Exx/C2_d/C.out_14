***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 04-Nov-2020  Start time: 11:10:05                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 18.000000 18.000000 18.000000 
FD_GRID: 18 18 18
FD_ORDER: 12
BC: D D D
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003675000
CHEB_DEGREE: 14
NSTATES: 6
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-04
TOL_POISSON: 1.00E-06
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/C2_d/C.out_14
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  1.000000 (Bohr)
Output printed to                  :  ../tests/Exx/C2_d/C.out_14
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.00 7.00 7.00
Number of atoms of type 1          :  2
Estimated total memory usage       :  34.32 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.5284387772E+00        8.964E-02        0.095
2            -3.5414068090E+00        5.590E-02        0.045
3            -3.5446384840E+00        2.368E-02        0.051
4            -3.5448530739E+00        5.345E-03        0.276
5            -3.5448596644E+00        2.912E-03        0.067
6            -3.5448681214E+00        1.002E-03        0.045
7            -3.5448685664E+00        2.381E-04        0.047
8            -3.5448687908E+00        1.235E-04        0.048
9            -3.5448701072E+00        1.555E-05        0.044
Total number of SCF: 9     
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.5448701072E+00 (Ha/atom)
Total free energy                  : -7.0897402145E+00 (Ha)
Band structure energy              : -3.6374584963E+00 (Ha)
Exchange correlation energy        : -3.1040201636E+00 (Ha)
Self and correction energy         : -3.6042996468E+01 (Ha)
Entropy*kb*T                       : -1.2802109632E-03 (Ha)
Fermi level                        : -2.5599629409E-01 (Ha)
Average force                      :  5.1683297699E+01 (Ha/Bohr)
Maximum force                      :  5.1683297699E+01 (Ha/Bohr)
Time for force calculation         :  0.034 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  0.995 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
