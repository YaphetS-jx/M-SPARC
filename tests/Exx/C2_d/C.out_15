***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 04-Nov-2020  Start time: 11:10:23                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 18.000000 18.000000 18.000000 
FD_GRID: 20 20 20
FD_ORDER: 12
BC: D D D
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003675000
CHEB_DEGREE: 14
NSTATES: 6
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-04
TOL_POISSON: 1.00E-06
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/C2_d/C.out_15
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.900000 (Bohr)
Output printed to                  :  ../tests/Exx/C2_d/C.out_15
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.20 7.20 7.20
Number of atoms of type 1          :  2
Estimated total memory usage       :  44.64 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.5174484368E+00        2.808E-01        0.103
2            -4.7779121177E+00        1.230E-01        0.058
3            -4.7925240454E+00        6.157E-02        0.068
4            -4.7956444028E+00        2.300E-02        0.067
5            -4.7960618033E+00        8.782E-03        0.065
6            -4.7961081812E+00        3.783E-03        0.064
7            -4.7961186344E+00        1.513E-03        0.059
8            -4.7961205059E+00        4.772E-04        0.061
9            -4.7961207130E+00        1.984E-04        0.059
10           -4.7961206185E+00        6.193E-05        0.063
Total number of SCF: 10    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -4.7961206185E+00 (Ha/atom)
Total free energy                  : -9.5922412370E+00 (Ha)
Band structure energy              : -4.0069798759E+00 (Ha)
Exchange correlation energy        : -3.5919354710E+00 (Ha)
Self and correction energy         : -3.4653893598E+01 (Ha)
Entropy*kb*T                       : -9.6403769749E-04 (Ha)
Fermi level                        : -3.2016098213E-01 (Ha)
Average force                      :  4.2242159293E+01 (Ha/Bohr)
Maximum force                      :  4.2242159293E+01 (Ha/Bohr)
Time for force calculation         :  0.038 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  0.969 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
