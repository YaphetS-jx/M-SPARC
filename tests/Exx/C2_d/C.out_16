***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 04-Nov-2020  Start time: 11:10:32                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 18.000000 18.000000 18.000000 
FD_GRID: 20 20 20
FD_ORDER: 12
BC: D D D
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003675000
CHEB_DEGREE: 14
NSTATES: 6
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-04
TOL_POISSON: 1.00E-06
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/C2_d/C.out_16
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.900000 (Bohr)
Output printed to                  :  ../tests/Exx/C2_d/C.out_16
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.20 7.20 7.20
Number of atoms of type 1          :  2
Estimated total memory usage       :  44.64 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.5174484368E+00        2.808E-01        0.102
2            -4.7779121177E+00        1.230E-01        0.058
3            -4.7925240454E+00        6.157E-02        0.067
4            -4.7956444028E+00        2.300E-02        0.070
5            -4.7960618033E+00        8.782E-03        0.077
6            -4.7961081812E+00        3.783E-03        0.068
7            -4.7961186344E+00        1.513E-03        0.059
8            -4.7961205059E+00        4.772E-04        0.057
Total number of SCF: 8     
1            -4.7820716585E+00        2.921E-02        117.928
2            -4.7827853517E+00        1.426E-02        34.658
3            -4.7830115795E+00        1.503E-03        34.807
4            -4.7830114849E+00        8.811E-04        34.826
5            -4.7830118211E+00        3.902E-04        34.876
6            -4.7830118775E+00        2.207E-04        38.035
