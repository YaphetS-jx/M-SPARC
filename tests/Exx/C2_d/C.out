***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 03-Nov-2020  Start time: 17:16:02                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 30.000000 30.000000 30.000000 
FD_GRID: 45 45 45
FD_ORDER: 12
BC: D D D
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003675000
CHEB_DEGREE: 15
NSTATES: 10
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/C2_d/C.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.666667 (Bohr)
Output printed to                  :  ../tests/Exx/C2_d/C.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  8.67 8.67 8.67
Number of atoms of type 1          :  2
Estimated total memory usage       :  408.61 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -1.0272515098E+01        2.154E+00        1.816
2            -1.0636967085E+01        1.296E+00        1.622
3            -1.0589488519E+01        3.069E-01        1.466
4            -1.0980167944E+01        3.350E-01        1.636
5            -1.1147784641E+01        1.561E-01        1.581
6            -1.1092266459E+01        1.917E-01        1.512
7            -1.0935922483E+01        6.079E-01        1.567
8            -1.1132944538E+01        2.322E-01        1.596
9            -9.4347249092E+00        8.418E-01        1.619
10           -9.7287907317E+00        8.299E-01        1.667
11           -1.1026868093E+01        2.047E-01        1.670
12           -1.0307121835E+01        6.987E-01        1.696
13           -1.0878046040E+01        6.126E-01        1.645
14           -1.0339895894E+01        8.211E-01        1.552
15           -1.0654251641E+01        7.690E-01        1.564
16           -1.0931284713E+01        6.189E-01        1.556
17           -1.0816096972E+01        6.977E-01        1.557
18           -9.6519495512E+00        8.894E-01        1.405
19           -1.0833824650E+01        6.808E-01        1.629
20           -1.0979639358E+01        5.217E-01        1.634
21           -8.5939904131E+00        8.962E-01        1.559
22           -9.9824990335E+00        7.531E-01        1.625
23           -1.0364194782E+01        7.954E-01        1.550
24           -1.1030914046E+01        1.577E-01        1.693
25           -1.1025375107E+01        4.362E-01        1.707
26           -1.0985182150E+01        5.395E-01        1.566
27           -1.1039830474E+01        3.894E-01        1.550
28           -1.1029818926E+01        4.420E-01        1.547
29           -1.0993709348E+01        5.400E-01        1.470
30           -1.0622873188E+01        7.715E-01        1.620
31           -9.3231647029E+00        8.462E-01        1.623
32           -9.8399947323E+00        8.687E-01        1.750
33           -9.8753988686E+00        8.719E-01        1.754
34           -1.0060689899E+01        8.599E-01        1.484
35           -1.0047187878E+01        8.606E-01        1.568
36           -1.0140160566E+01        8.523E-01        1.370
37           -1.0762896102E+01        7.154E-01        1.517
38           -1.0801090560E+01        6.786E-01        1.606
39           -1.0635265904E+01        7.546E-01        1.564
40           -1.0508001428E+01        4.739E-01        1.551
41           -1.0955966005E+01        4.757E-01        1.646
42           -1.0356157401E+01        7.973E-01        1.631
43           -1.0322144633E+01        8.052E-01        1.488
44           -1.0162899958E+01        8.243E-01        1.429
45           -1.0092007065E+01        8.245E-01        1.562
46           -1.0066506810E+01        8.302E-01        1.400
47           -1.0439236905E+01        7.263E-01        1.408
48           -1.0779345865E+01        3.176E-01        1.565
49           -1.0984450922E+01        1.767E-01        1.642
50           -1.1018760563E+01        1.742E-01        1.555
51           -1.1049707405E+01        2.530E-01        1.644
52           -1.1034391205E+01        4.243E-01        1.569
53           -1.1013492140E+01        4.730E-01        1.559
54           -1.1030253933E+01        4.465E-01        1.489
55           -1.1047962203E+01        4.385E-01        1.493
56           -1.1021935402E+01        3.632E-01        1.421
57           -1.1053849538E+01        1.814E-01        1.557
58           -9.1687662779E+00        8.542E-01        1.640
59           -1.0919654025E+01        1.869E-01        1.638
60           -9.8435288814E+00        8.732E-01        1.669
61           -1.0319733949E+01        8.280E-01        1.700
62           -1.0463550027E+01        8.031E-01        1.586
63           -1.0539758247E+01        7.867E-01        1.573
64           -1.0444549716E+01        8.059E-01        1.488
65           -9.9107667621E+00        8.332E-01        1.485
66           -1.1059175963E+01        1.667E-01        1.557
67           -1.0918431483E+01        5.349E-01        1.711
68           -8.6049011394E+00        8.937E-01        1.596
69           -1.1025339459E+01        4.544E-01        1.769
70           -1.1048184609E+01        3.901E-01        1.828
71           -1.1017684659E+01        4.794E-01        1.598
72           -1.1021398289E+01        4.710E-01        1.750
73           -1.0852757436E+01        6.320E-01        1.498
74           -1.0596592609E+01        4.946E-01        1.699
75           -1.1007361482E+01        4.251E-01        1.728
76           -9.7392800048E+00        8.626E-01        1.717
77           -1.0285145105E+01        8.034E-01        1.687
78           -1.0520320005E+01        7.292E-01        1.625
79           -1.1032990215E+01        3.848E-01        1.630
80           -1.0771028090E+01        7.092E-01        1.709
81           -1.0779656416E+01        7.071E-01        1.617
82           -1.0764106740E+01        7.106E-01        1.422
83           -9.8470989063E+00        8.680E-01        1.496
84           -8.7321141754E+00        8.862E-01        1.606
85           -1.1032726523E+01        3.636E-01        1.688
86           -1.0784259538E+01        6.890E-01        1.733
87           -1.0690587839E+01        7.161E-01        1.578
88           -1.1066453417E+01        1.335E-01        1.550
89           -1.1097858479E+01        3.024E-01        1.675
90           -1.1132852035E+01        1.006E-01        1.683
91           -1.1144233670E+01        8.933E-02        1.650
92           -1.1137027759E+01        1.711E-01        1.474
93           -1.0925641562E+01        3.834E-01        1.500
94           -1.1143102953E+01        6.025E-02        1.575
95           -1.1135994991E+01        1.449E-01        1.646
96           -1.0906980023E+01        4.401E-01        1.520
97           -1.1094084003E+01        1.055E-01        1.570
98           -1.1118615693E+01        2.775E-01        1.699
99           -1.1048975917E+01        4.450E-01        1.672
100          -1.1044240807E+01        4.528E-01        1.532
Total number of SCF: 100   
